sql/create_train_val.go

```diff
@@ -156,13 +156,20 @@ func createTable(database, table, origin string, db *DB, cond string) (string, e
 }
 
 func namingTrainAndValDataset(origTable string) *trainAndValDataset {
-	// hive returns a table with a database name
-	flattenTbl := strings.Replace(origTable, ".", "__", -1)
+	// NOTE: We always create temporary table under current in using
+	// database or MaxCompute project, because the origin table may
+	// from another database which we may not have write access.
+	var origTableWithNoDBPrefix string
+	origTableParts := strings.Split(origTable, ".")
+	// FIXME(typhoonzero): use last part, if len != 2 should return some error
```

Comment:
I'm not sure if we can make this assertion. Is it possible the `len` equals to 1?

Reply:
We must support user write sqls like `select * from myproject.mytabe train ...`

Reply:
Will update the comment, thanks.